addons/knowledge/static/tests/tours/knowledge_main_flow_tour.js

```diff
@@ -0,0 +1,185 @@
+/** @odoo-module */
```

Comment:
kind of disappointed at the fact that you only have 180loc of tests, and in a tour, no unit tests at all...

Reply:
Side note for tests: we have a lot of tests for permission / security mechanism in python, this took a lot of our time devoted to tests :D . We will do the followup and add tests for the JS before 15.4 freeze (both functional and unit tests); and anyway this will be necessary to ensure convert to owl is done neatly.